.circleci/config.yml

```
@@ -248,6 +249,20 @@ jobs:
           command: |
             aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
+            aws s3 sync "s3://login-gov-test-coverage/coverage-artifacts/$CIRCLE_BUILD_NUM" coverage/
+            bundle exec spec/simplecov_merger.rb
+            mkdir coverage_summary
+            mv coverage/index.html coverage_summary/
+            mv coverage/assets coverage_summary/
+            mv coverage/.resultset.json coverage_summary/
+            aws s3 --region us-west-2 rm s3://login-gov-test-coverage/coverage-artifacts/$CIRCLE_BUILD_NUM --recursive
+      - store_artifacts:
+          path: coverage_summary/index.html
+      - store_artifacts:
+          path: coverage_summary/.resultset.json
+      - store_artifacts:
+          path: coverage_summary/assets
```

It looks like `coverage_summary` already only contains the files we care about, so can we just upload the entire dir?

```suggestion
      - store_artifacts:
          path: coverage_summary/
```